### PR TITLE
RHDEVDOCS-6270: Release notes PR for GitOps 1.14.2 z-stream release

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.14

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6270

Link to docs preview:

[Release Notes for Red Hat OpenShift GitOps 1.14.2](https://85102--ocpdocs-pr.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes-1-14.html#gitops-release-notes-1-14-2_gitops-release-notes)

QE review:
- [x] QE has approved this change.

SME review: @ishitasequeira @keithchong 
Peer review: @skrthomas 
QE review: @varshab1210

Additional information:

### CODE REVIEW

File: modules/gitops-release-notes-1-14-2.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-14-2_{context}"]
+== Fixed issues
```
Comment (ishitasequeira) [inline modules/gitops-release-notes-1-14-2.adoc:32]: @Dhruv-Soni11 , we also need to add https://issues.redhat.com/browse/GITOPS-3465 to 1.14.2 release notes.

Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-2.adoc:32]: Fixed it now @ishitasequeira. Please take a look after some time when the build is ready.

**Comment by keithchong** [comment]
@Dhruv-Soni11 , the 1.14.2 RN entry for 5788 looks good. 

"Before this update, the New App panel behaved unexpectedly as it reopened immediately after being closed, and typing in fields causes the cursor and text to flash repeatedly. With this fix, the New App panel operates as intended. [GITOPS-5788](https://issues.redhat.com/browse/GITOPS-5788)"

One question I have is whether to reference 5788 or 5671?  There were duplicates opened in Jira.

**Comment by Dhruv-Soni11** [comment]
> @Dhruv-Soni11 , the 1.14.2 RN entry for 5788 looks good.
> 
> "Before this update, the New App panel behaved unexpectedly as it reopened immediately after being closed, and typing in fields causes the cursor and text to flash repeatedly. With this fix, the New App panel operates as intended. [GITOPS-5788](https://issues.redhat.com/browse/GITOPS-5788)"
> 
> One question I have is whether to reference 5788 or 5671? There were duplicates opened in Jira.

@keithchong - I don't know the answer to this question so I request @ishitasequeira to help us regarding this.

File: modules/gitops-release-notes-1-14-2.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+* Before this update, the New App panel behaved unexpectedly as it reopened immediately after being closed, and typing in fields causes the cursor and text to flash repeatedly. With this fix, the New App panel operates as intended. link:https://issues.redhat.com/browse/GITOPS-5788[GITOPS-5788]
```
Comment (ishitasequeira) [inline modules/gitops-release-notes-1-14-2.adoc:34]: As mentioned in the slack thread https://redhat-internal.slack.com/archives/C06SPJE651R/p1731999243234959. We could remove this one

File: modules/gitops-release-notes-1-14-2.adoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-2_{context}"]
+= Release notes for {gitops-title} 1.14.2
+
+{gitops-title} 1.14.2 is now available on {OCP} 4.13, 4.14, 4.15, 4.16, and 4.17.
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-14-2.adoc:10]:
```suggestion
{gitops-title} 1.14.2 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, 4.16, and 4.17.
```

File: modules/gitops-release-notes-1-14-2.adoc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+[id="errata-updates-1-14.2_{context}"]
+== Errata updates
+
+[id="RHBA-XXXX:NNNN-gitops-1-14-2-security-update-advisory_{context}"]
+=== RHBA-XXXX:NNNN - {gitops-title} 1.14.2 security update advisory
+
+Issued: 2024-11-11
```
Comment (varshab1210) [inline modules/gitops-release-notes-1-14-2.adoc:18]: Please change the date once we ship

**Comment by ishitasequeira** [comment]
> GITOPS-5788

As we marked https://issues.redhat.com/browse/GITOPS-5788 as a dupe. Please refer https://issues.redhat.com/browse/GITOPS-5671

File: modules/gitops-release-notes-1-14-2.adoc (hunk 2 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-2.adoc:34]: Fixed, as per this comment I am removing this fixed issue from the content.

File: modules/gitops-release-notes-1-14-2.adoc (hunk 3 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-2.adoc:10]: Fixed.

File: modules/gitops-release-notes-1-14-2.adoc (hunk 4 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-2.adoc]: Fixed.

**Comment by varshab1210** [comment]
LGTM, thanks

**Review by ishitasequeira — APPROVED** [review]
LGTM

**Review by skrthomas — COMMENTED** [review]
/LGTM

File: modules/gitops-release-notes-1-14-2.adoc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+----
+
+[id="fixed-issues-1-14-2_{context}"]
+== Fixed issues
+
+* Before this update, the New App panel behaved unexpectedly as it reopened immediately after being closed, and typing in fields caused the cursor and text to flash repeatedly. With this fix, the New App panel operates as intended. link:https://issues.redhat.com/browse/GITOPS-5671[GITOPS-5671]
+
+* Before this update, when using the dynamic scaling feature, application controller pods consumed high memory and clusters were not distributed properly across shards. This update ensures that clusters are distributed across shards. https://issues.redhat.com/browse/GITOPS-3465[GITOPS-3465]
++
+For more information, see xref:../declarative_clusterconfig/sharding-clusters-across-argo-cd-application-controller-replicas.adoc#gitops-argo-cd-dynamic-scaling-in-web-console_sharding-clusters-across-argo-cd-application-controller-replicas[Enabling dynamic scaling of shards in the web console section]
```
Comment (bergerhoffer) [inline modules/gitops-release-notes-1-14-2.adoc:38]:
```suggestion
For more information, see xref:../declarative_clusterconfig/sharding-clusters-across-argo-cd-application-controller-replicas.adoc#gitops-argo-cd-dynamic-scaling-in-web-console_sharding-clusters-across-argo-cd-application-controller-replicas[Enabling dynamic scaling of shards in the web console].
```

**Review by bergerhoffer — COMMENTED** [review]
This PR has 2 commits, which need to be squashed before it can be merged.

Also, can you please fix the one other comment while you're doing that? Thanks!

File: modules/gitops-release-notes-1-14-2.adoc (hunk 5 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-1-14-2.adoc]: Fixed.

**Comment by Dhruv-Soni11** [comment]
Hi @bergerhoffer I have incorporated the feedback for your comment and squashed the extra commit. PTAL and merge the PR if everything looks fine.

**Pull request merged**